quickstarts/kit/custom-istio-enricher/pom.xml

```diff
@@ -56,9 +56,9 @@
         <scope>import</scope>
       </dependency>
       <dependency>
-        <groupId>me.snowdrop</groupId>
+        <groupId>io.fabric8</groupId>
         <artifactId>istio-client</artifactId>
-        <version>${istio-client.version}</version>
+        <version>5.12.2</version>
```

Comment:
Could you please replace the current variable value in L41 `<istio-client.version>1.7.5-Beta2</istio-client.version>` and still use it here?

Reply:
I have just committed this change now, also I very sorry about not  commenting on the issue or asking to be assigned on it before my PR